### PR TITLE
WIP: switch nav according to login

### PR DESCRIPTION
**Pull request by harrier91**

### CODE REVIEW

File: src/pages/common/Header/CommunityHeader/CommunityHeader.tsx [hunk 1]
```diff
@@ -139,16 +139,14 @@ export class CommunityHeader extends React.Component<IProps, IState> {
...
-            <IconButton component="span">
-              <MailOutlinedIcon />
-            </IconButton>
-            <IconButton component="span">
-              <MdNotifications />
-            </IconButton>
-          </div>
-          {this.props.userStore.user ? (
-            <>
+          {this.props.userStore.user ?
```
Comment (tx4x) [inline src/pages/common/Header/CommunityHeader/CommunityHeader.tsx]: not sure what's best practice but I'd prefer this as { this.props.userStore.user && React.Fragment }. Eventually this is also better written as extracted functional component, the render function became a little unreadable :-)

Comment (harrier91) [reply src/pages/common/Header/CommunityHeader/CommunityHeader.tsx]: But there are 2 options in this case, either the Menu or the Login button, that why I could use the &&

File: src/pages/Home/Home.scss [hunk 2]
```diff
@@ -27,6 +27,15 @@ $colorSecondary: "yellow";
   margin: auto;
 }
 
+.login-btn {
```
Comment (BenGamma) [inline src/pages/Home/Home.scss]: I missed this .scss file during last refactor, my bad. We are not styling that way anymore, please use the dediacted component`src/components/Button` instead
More infos in the wiki about styling guidelines https://github.com/OneArmyWorld/onearmy/wiki/Frontend

**Comment by BenGamma** [comment]
Hey tks for the PR,
can you point me from which design did you based your styling ?

**Comment by harrier91** [comment]
Hi,
I was following this: https://sketch.cloud/s/KGerV/VrVvaPw/play

**Comment by BenGamma** [comment]
As discussed on slack :
* please add a more precise description of your changes and add related issue number
* I created a new branch `feature/global-site-navbar` to make the PR to, instead of master

Thanks by the way ! and sorry for inconvenience

**Pull request closed without merging**